app/javascript/gobierto_data/webapp/components/sets/data/SQLEditorHeader.vue

```diff
@@ -36,25 +36,31 @@
     </div>
 
     <div class="gobierto-data-sql-editor-container">
-      <Button
-        v-clickoutside="closeQueriesModal"
-        :class="{ 'remove-label' : removeLabelBtn }"
-        :title="labelButtonQueries"
-        class="btn-sql-editor"
-        icon="list"
-        color="var(--color-base)"
-        background="#fff"
-        @click.native="isQueriesModalActive = !isQueriesModalActive"
-      />
+      <!-- //Closable directive doesn't work with component, so i've used the html attribute :\ -->
```

Comment:
Events doesn't work on Component unless you set `.native`, have you tried?

Reply:
Yes, I tried, the problem isn't with the component, it's with the closable directive